setup.py

```diff
@@ -107,7 +103,6 @@ def run(self):
     "mock>=3.0.0",
     "xmltodict>=0.11.0",
     "awscli>=1.16.125",
-    "google-compute-engine",
```

Comment:
Btw, I think I remember adding this one because of binary built by pyinstaller, when using it with gs remote. Can we try rather fixing the version of this package instead of removing it? Just to be safe.

Reply:
hmm, is it possible to check which version will be the "good" one? Or latest should be enough?